types/vuex-i18n/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
For new package, there should only be `"extends": "dtslint/dt.json"` in `tslint.json` file